colcon_core/location.py

```diff
@@ -130,23 +134,33 @@ def create_log_path():
         # create latest symlink
         if sys.platform == 'win32':
             return
```

Comment:
Nitpick: Not sure if it's worth checking that twice. I guess you want to check it in the function itself in case you want to use it in other places.

Reply:
I removed the check from the function. Since it is private it is not supposed to be used from other places anyway.